docs/Your-first-app-with-System-CommandLine.md

```diff
@@ -13,6 +13,13 @@ Open a new console and run the following commands:
 
 ## Install the System.CommandLine package
 
+Use dotnet tool to install the package
```

Comment:
I think this would be a little simpler just to add this as a third command in the section just above.

Reply:
It felt more correct to have this under the heading "Install the System.CommandLine package", rather than under "Create a new console app". But should I move it upwards?